### PR TITLE
dev, 3.0: Fix dead link, coordinate helm link, replace hard-coded chart version

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

### What is changed, added or deleted? 
Fix dead link, coordinate helm link, replace hard-coded chart version

### Which version does your change affect? 
`dev` `3.0`

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/reference/tidb-operator-overview.md [hunk 1]
```diff
@@ -39,14 +39,14 @@ TiDB Operator 提供了多种方式来部署 Kubernetes 上的 TiDB 集群：
...
+    - 公有云：参考 [AWS 部署文档](how-to/deploy/tidb-in-kubernetes/aws-eks.md)、[GCP 部署文档](how-to/deploy/tidb-in-kubernetes/gcp-gke.md)或[阿里云部署文档](how-to/deploy/tidb-in-kubernetes/alibaba-cloud.md)在对应的公有云上一键部署生产可用的 TiDB 集群并进行后续的运维管理；
+    - 现有 Kubernetes 集群：首先按照[部署 TiDB Operator](how-to/deploy/tidb-in-kubernetes/tidb-operator.md)在集群中安装 TiDB Operator，再根据[在标准 Kubernetes 集群上部署 TiDB 集群](how-to/deploy/tidb-in-kubernetes/general-kubernetes.md)来部署你的 TiDB 集群。对于生产级 TiDB 集群，你还需要参考 [TiDB 集群环境要求](reference/configuration/tidb-in-kubernetes/local-pv-configuration.md)调整 Kubernetes 集群配置并根据[本地 PV 配置](reference/configuration/tidb-in-kubernetes/local-pv-configuration.md)为你的 Kubernetes 集群配置本地 PV，以满足 TiKV 的低延迟本地存储需求。
 
 在任何环境上部署前，都可以参考 [TiDB 集群配置](reference/configuration/tidb-in-kubernetes/cluster-configuration.md)来自定义 TiDB 配置。
 
 部署完成后，你可以参考下面的文档进行 Kubernetes 上 TiDB 集群的使用和运维：
 
-+ [管理 TiDB 集群](how-to/maintain/tidb-in-kubernetes/tidb-cluster.md)
++ [管理 TiDB 集群](how-to/maintain/tidb-in-kubernetes/cluster-configuration.md)
```
Comment (dcalvin) [inline dev/reference/tidb-operator-overview.md]: I remember we should already have removed this doc

Comment (aylei) [reply dev/reference/tidb-operator-overview.md]: Good catch

**Comment by aylei** [comment]
@dcalvin I've update the document, PTAL again

File: dev/how-to/troubleshoot/tidb-in-kubernetes.md [hunk 2]
```diff
@@ -258,4 +258,4 @@ kubectl logs -f <tidb-pod-name> -n <namespace>
 
     * 检查 node 上的相关 route 规则是否正确
     * 检查网络插件服务是否正常
-    * 参考上面的[Pod 之间网络不通](#Pod-之间网络不通)章节
+    * 参考上面的[Pod 之间网络不通](#pod-之间网络不通)章节
```
Comment (xiaojingchen) [inline dev/how-to/troubleshoot/tidb-in-kubernetes.md]:
```suggestion
    * 参考上面的[Pod 之间网络不通](#Pod-之间网络不通)章节
```

File: v3.0/reference/tools/tools-in-kubernetes.md [hunk 3]
```diff
@@ -251,6 +251,6 @@ helm del --purge <release-name>
 
 ## 使用 Terraform
 
-[Terraform](https://www.terraform.io/) 是一个基础设施即代码管理工具。它允许用户使用声明式的风格描述自己的基础设施，并针对描述生成执行计划来创建或调整真实世界的计算资源。Kubernetes 上的 TiDB 使用 Terraform 来在公有云上创建和管理 TiDB 集群。
+[Terraform](https://www.terraform.io/) 是一个基础设施即代码（Infrastructore as Code）管理工具。它允许用户使用声明式的风格描述自己的基础设施，并针对描述生成执行计划来创建或调整真实世界的计算资源。Kubernetes 上的 TiDB 使用 Terraform 来在公有云上创建和管理 TiDB 集群。
```
Comment (dcalvin) [inline v3.0/reference/tools/tools-in-kubernetes.md]:
```suggestion
[Terraform](https://www.terraform.io/) 是一个基础设施即代码（Infrastructure as Code）管理工具。它允许用户使用声明式的风格描述自己的基础设施，并针对描述生成执行计划来创建或调整真实世界的计算资源。Kubernetes 上的 TiDB 使用 Terraform 来在公有云上创建和管理 TiDB 集群。
```

**Pull request merged**